assets/shaders/viewport_render.wgsl

```diff
@@ -279,27 +279,27 @@ fn dda_step_to_next_sibling(
     return result;
 }
 
-const key_none_value : u32 = 4294967295u;
-
 // Unique to this implementation, not adapted from rust code, corresponds to:
-// crate::octree::raytracing::classic_raytracing_on_bevy_wgpu::meta_set_is_leaf
+//crate::octree::raytracing::classic_raytracing_on_bevy_wgpu::meta_set_is_leaf
 fn is_leaf(sized_node_meta: u32) -> bool {
     return 0 < (0x01000000 & sized_node_meta);
 }
 
 // Unique to this implementation, not adapted from rust code, corresponds to:
-// crate::octree::raytracing::classic_raytracing_on_bevy_wgpu::meta_set_lvl2_occupancy_bitmask
+//crate::octree::raytracing::classic_raytracing_on_bevy_wgpu::meta_set_lvl2_occupancy_bitmask
 fn get_lvl2_occupancy_bitmask(sized_node_meta: u32) -> u32 {
     return (0x000000FF & sized_node_meta);
 }
 
-// Unique to this implementation, not adapted from rust code
+// Functionality-wise this function is more generic, than its coutnerpart
+// and is used in voxel matrix mapping too
+//crate::spatial::math::bitmask_mapping
 fn voxel_matrix_index_mapping(i: vec3u, dimensions: vec2u) -> u32 {
     return (i.x + (i.y * dimensions.y) + (i.z * dimensions.x * dimensions.y));
 }
 
-// crate::spatial::math::is_bitmask_occupied_at_octant
-fn is_bitmask_occupied_at_octant(bitmask: u32, octant: u32) -> bool {
+//crate::spatial::math::is_bitmap_occupied_at_octant
+fn is_bitmap_occupied_at_octant(bitmask: u32, octant: u32) -> bool {
```

Comment:
Just something I noted: `bitmask` and `bitmap` are used interchangeably... maybe we should stick with one term? I mean here the method is called `is_bitmap_occupied_at_octant` but the parameter is named `bitmask` AND one function down is called `position_in_bitmask_64bits`.

Nothing really important, just something that I noted because it was changed

Reply:
The main difference between the 2 is while a bitmap is a field of binary numbers, a bitmask is a value that is used in filtering bitmap values; I tried to name them as such, but the difference is really subtle